test/src/unit-backwards_compat.cc

```diff
@@ -1364,3 +1364,72 @@ TEST_CASE(
   vfs.remove_dir(get_commit_dir(array_read2.uri()));
   vfs.remove_dir(schema_folder);
 }
+
+TEST_CASE(
```

Comment:
nit: the current tests in this file are used with https://github.com/TileDB-Inc/TileDB-Unit-Test-Arrays (exclusively, I think), so it would be clearer to put this in `test/regression`. We are checking this in along with the example array _as_ a regression test; strictly speaking, the "backward compatibility" aspect should be handled by adding generation of this array variant to the compat generator and checking it in as a compatibility example (sc-33656).

Reply:
Moved.

Reply:
Thanks!